Add manual page

wittejm opened the pull request:
This is the feature branch for adding the Manual page.

While @hmarcks and I work on the html / React pieces respectively, we can maintain it has a regular shared branch with individual pull requests rather than pushing to it directly.

wittejm commented:
@hmarcks though I did make one more commit just now, to move the header code that was in the Landing page into the shared Header component, so that it can get reused on the Manual page. Hopefully that doesn't interfere with your local changes too much.

hmarcks commented:
@wittejm sounds good, I'm actually just working from an HTML file I created on my desktop so it's not even local yet :-)

hmarcks commented:
Actually just finishing up docs now...

hmarcks commented:
Okay good to go

hmarcks commented on src/frontend/src/components/Manual/index.tsx:
```diff
@@ -0,0 +1,984 @@
+          </section>
+          <section className="bt bw1 b--light-gray pt2 mb5">
+            <h2 className="f2 fw9 mb3" id="review">
+              Further Analysis Needed
```
Just to confirm, if Michael agrees, all of them can be removed?

Then we don't have any scenarios where the charge needs a review and doesn't have an input to clarify it?

wittejm replied:
I checked with Michael and he agreed we can delete this section.

wittejm commented:
I'm about to push these minor changes

hmarcks commented:
Do you mind if I do? I have some other updates so would like to avoid conflicts

wittejm commented:
sure, sounds good

wittejm commented:
Ok. I rebased (and meanwhile borked then un-borked the commit history), and added one last React logic adjustment to the Header (with no change in behavior). I can ask @KentShikama to look over the react code but otherwise if you're satisfied I can merge

hmarcks commented:
Good on my end, thanks!

KentShikama commented on src/frontend/src/components/App/index.tsx:
```diff
@@ -1,14 +1,13 @@
     return (
       <Authenticator>
-        <Header />
+        {/*<Header /> // added to individual pages and excluded from others */}
```
What is this comment?
